genai-quickstart-pocs-python/amazon-bedrock-intelligent-document-processing-poc/pages/2_🔎_Step_2:_Analyze_Document_with_Textract.py

```diff
@@ -0,0 +1,43 @@
+import streamlit as st
+import os
+from idp.s3_utils import list_files_in_s3
+from idp.textract_utils import process_document
+from dotenv import load_dotenv
+
+# Load environment variables
+load_dotenv()
+
+# Get the bucket name from the .env file
+bucket_name = os.getenv("save_folder")
+
+# Streamlit app title
+st.title(f""":rainbow[Analyze Document with Amazon Textract]""")
+
```

Comment:
Can you put a little one or two sentence blurb about what this part of the POC is doing?
What are the key/value pairs?